Formula/c.rb

```diff
@@ -0,0 +1,17 @@
+class C < Formula
+  desc 'Compile and execute C "scripts" in one go!'
+  homepage "https://github.com/ryanmjacobs/c"
+  url "https://github.com/ryanmjacobs/c/archive/v0.14.tar.gz"
+  sha256 "2b66d79d0d5c60b8e6760dac734b8ec9a7d6a5e57f033b97086821b1985a870b"
+  license "MIT"
+  head "https://github.com/ryanmjacobs/c.git", branch: "master"
+
+  def install
+    bin.install "c"
+  end
+
+  test do
+    assert_match "Execute C programs from the command line.", \
+      shell_output("#{bin}/c 2>&1", 1)
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
OKay.